pkg/chunked/compressor/compressor.go

```diff
@@ -6,13 +6,16 @@ package compressor
 
 import (
 	"bufio"
+	"bytes"
 	"encoding/base64"
 	"io"
 
 	"github.com/containers/storage/pkg/chunked/internal"
 	"github.com/containers/storage/pkg/ioutils"
 	"github.com/opencontainers/go-digest"
 	"github.com/vbatts/tar-split/archive/tar"
+	"github.com/vbatts/tar-split/tar/asm"
+	"github.com/vbatts/tar-split/tar/storage"
```

Comment:
This is, sadly, going to be added into every single c/image/copy and c/image/docker user…

I don’t immediately see a way to avoid that, I’m afraid. But if anyone else does see a way, I’d love that.